### ISSUE TITLE
Add the ability to "open note of the day in 1 click".

### ISSUE THREAD

**Issue by Fertion** [ozntel/oz-calendar#25]
Hi, thanks for this plugin.

I (and I think many users too) have only 1 relevant note per day. What is missing is an option that would allow clicking on a date to immediately open the note for that day, provided that there is only 1 note for that day.

Now you have to make two clicks, first on the day and then select this note under the calendar.

**Comment by Bandit** [comment]
Yeah I'd like an option where we can fill the sidebar with as many months as can fit, and a click on a day takes you to the corresponding entry for that day. I use my Obsidian for daily journalling so only have 1 entry per day in the format `YYYY/MMMM/DD` (e.g. `/2024/April/16`)
